packages/common/index.ts

```diff
@@ -1136,26 +1097,15 @@ export class QuestionTypeParams {
   queueId?: number
 }
 
-export class QuestionTypeParamsWithOptionalQueueId {
-  @IsInt()
-  @IsOptional()
-  id?: number
-
-  @IsInt()
-  @IsOptional()
-  cid?: number
-
-  @IsString()
-  name!: string
-
-  @IsString()
-  @IsOptional()
-  color?: string
-
-  @IsInt()
-  @IsOptional()
-  queueId?: number
+// named QuestionTypeType to not conflict with the UI component QuestionType
```

Comment:
I feel like we should rename this type, it's more confusing than it was before

Reply:
yeah it's not ideal for sure lol.
Ideally, we could rename all instances of questionType into questionTag or something like that (since `Type` gets confused with our types and sounds more mutually-exclusive than `tag`), but that's not worth the effort atm imo.

I mean it'd probably be better just to rename the type into QuestionType and then rename the component that displays a questionType, but meh.